x-pack/plugins/fleet/public/services/get_template_url_from_agent_policy.ts

```diff
@@ -7,21 +7,19 @@
 
 import type { AgentPolicy } from '../types';
 
```

Comment:
Great idea on reusing this function, I think we can export from this file an object with the supported `templateUrlFieldName`:

```js
export const SUPPORTED_TEMPLATES = {
      CLOUD_FORMATION: 'cloud_formation_template_url',
      ARM: 'arm_template_url'
}
```

this way later we can reuse it like this and avoid typos:

```js
import { getTemplateUrlFromAgentPolicy, SUPPORTED_TEMPLATES } from '../../services';

const cloudFormationTemplateFromAgentPolicy = getTemplateUrlFromAgentPolicy(
      SUPPORTED_TEMPLATES.CLOUD_FORMATION,
      agentPolicy
    );
```